ironfish-cli/src/commands/service/bridge/deposit.ts

```diff
@@ -175,4 +197,9 @@ export class Deposit extends IronfishCommand {
 
     this.log(`Deposit transaction hash: ${sendResponse.content.hash}`)
   }
+
+  encodeEthAddress(address: string): string {
+    const buffer = Buffer.from(address, 'hex')
+    return buffer.toString('binary')
```

Comment:
This doesn't seem to work. This memo is converted to a Buffer with utf8 encoding which results in a buffer of 34 bytes. The last two bytes get truncated from the memo that ends up on the note